.github/workflows/repo-request.yml

```diff
@@ -1,29 +1,53 @@
-name: Create new repo for repo request 
+name: Create new repo for repo request
 
 on:
- issues:
+  issues:
     types: [labeled]
-    
+  workflow_dispatch:
```

Comment:
nice one 👍 can you add an exmaple json as default? default: ....

Reply:
kept it empty for a reason: whenever the issue template changes this default value would have to change as well and you can not check if its done until it fails

would try to automate this in a separate workflow asap

what do you think?